### PR TITLE
[MAP] Adds fallback to original value when input not found

### LINKED ISSUE #10092: [MAP transformation] Pass on missing string
Hello,

I translate incoming strings with the MAP transformation in another language. When I have in the .map-File no translation for an incoming string, the string is empty on OpenHAB and I have the following message in the log:
Could not transform 'xxx' with the file 'de.map' : Target value not found in map for 'xxx'.
The originate string would be better than none at all. I have two ideas to fix the issue:

Idea 1:
In the map-file add the default-entry with %s and resolve the placeholder. Example:
`=%s`

Idea 2:
In the "Incoming Value Transformations"-section choose a fallback.
Example: `MAP:de.map:%s` instead of `MAP:de.map`

Thanks

### PR DESCRIPTION
**Pull request by clinique**
Resolves #10092

### CODE REVIEW

File: bundles/org.openhab.transform.map/README.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ This file should be in property syntax, i.e. simple lines with "key=value" pairs
 The file format is documented [here](https://docs.oracle.com/javase/8/docs/api/java/util/Properties.html#load-java.io.Reader-).
 To organize the various transformations one might use subfolders.
 
-A default value can be provided if no matching entry is found by using "=value" syntax
+A default value can be provided if no matching entry is found by using "=value" syntax. 
+Defining this default value using `%s` would then return the non transformed input string.
```
Comment (lolodomo) [inline bundles/org.openhab.transform.map/README.md:11]: %s looks a little technical for a final user and not intuitive at all?

Comment (lolodomo) [reply bundles/org.openhab.transform.map/README.md:11]: Maybe `_source_` ?

**Comment by jlaur** [comment]
@clinique, if you change "Solves" to "Resolves" in the PR description, the issue will be automatically linked and closed upon PR completion.

File: bundles/org.openhab.transform.map/README.md (hunk 1 above)
Comment (clinique) [reply bundles/org.openhab.transform.map/README.md:11]: Ok, good idea. Let's do that.

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
I see no real risk with that change, the probability that someone defined `_source_` is very low I believe.

**Pull request merged**